flutter/scripts/generate-jni-bindings.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [[ -n ${CI:+x} ]]; then
+    echo "Running in CI – setting up Flutter SDK first"
+    # Note: keep version in sync with other binding generation scripts.
+    curl -Lv https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.27.3-stable.tar.xz --output /tmp/flutter.tar.xz
```

Comment:
Shouldn't we rely on the CI action to setup flutter? Here we pin to a specific version, when in CI we use the most recent one.

Reply:
I'm not sure if that will work with the updater workflow, I think in that case we'd need to update the upstream workflow action to allow us to inject steps before executing the script

fwiw it's also done the same way in `generate-cocoa-bindings.sh`